nbbuild/templates/projectized.xml

```diff
@@ -123,7 +123,7 @@
         </condition>
     </target>
     <target name="-init-bootclasspath-prepend-run9" depends="-init-bootclasspath-prepend-compile" if="have-jdk-1.9">
-        <condition property="test.bootclasspath.prepend.args" value="--limit-modules=java.base,java.logging,java.xml,java.prefs,java.desktop,java.management,java.instrument,jdk.zipfs,java.scripting,java.naming,jdk.jdi">
+        <condition property="test.bootclasspath.prepend.args" value="--limit-modules=java.base,java.logging,java.xml,java.prefs,java.desktop,java.management,java.instrument,jdk.zipfs,java.scripting,java.naming,jdk.jdi,jdk.unsupported">
```

Comment:
excellent! This means we have another job which could run on JDK 11+ (e.g in a matrix).

Reply:
Not really - there are still some failing tests, I asked @dbalek  for an assistance as the failure seems to be deeply burried in javac binding infrastructure. But it's definitely a step forward.